cypress/support/pages/login_management/pageLoginManagement.js

```diff
@@ -183,17 +186,54 @@ class Login_Management {
 	}
 
 	clickOnForgotPassword() {
-		cy.get(Login_Management.#passwordRecoveryButton).click();
+		cy.get("body").then((body) => {
+			if (body.find(Login_Management.#buttonLoginViaEmailNbc).length > 0) {
+				// If it exists in case of NBC login page, click on it and then the second element
+				cy.get(Login_Management.#buttonLoginViaEmailNbc)
+					.first()
+					.click()
+					.then(() => {
+						// locate the button in the specific class as they are used in multiple places
+						const pwdRecoveryButtonSelector =
+							".email-login-section " +
+							Login_Management.#passwordRecoveryButton;
+						cy.get(pwdRecoveryButtonSelector).click();
+					});
+			} else {
```

Comment:
we don't need else block as if block suffix it. here is the pseudo code for it
```js
cy.get("body").then((body) => {
			if (body.find(Login_Management.#buttonLoginViaEmailNbc).length) {
				cy.get(Login_Management.#buttonLoginViaEmailNbc).first().click();
			}
			cy.get(Login_Management.#passwordRecoveryButton).first().click();
		});

Reply:
Using 'else' is required here, because in case of NBC password recovery button must be clicked within the parent class to avoid multiple element error, otherwise dbc & brb directly execute in the 'else' block.

Reply:
done!